src/System.Diagnostics.StackTrace/tests/StackTraceSymbolsTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using Xunit;
+
+namespace System.Diagnostics.SymbolStore.Tests
+{
+    public class StackTraceSymbolsTests
+    {
+        [Fact]
+        public void StackTraceSymbolsDoNotLockFile()
+        {
+            var asmPath = typeof(StackTraceSymbolsTests).Assembly.Location;
+            var pdbPath = Path.ChangeExtension(asmPath, ".pdb");
+
+            Assert.True(File.Exists(pdbPath));
```

Comment:
This assumes the test project builds with non-embedded PDB, which might not be true. CoreFX might be overriding Arcade SDK defaults, but Arcade uses embedded PDBs for local and PR validation builds. See https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/ProjectDefaults.props#L25-L38

Reply:
I felt so clever coming up with this very simple test 😄 .